ts/Core/Tooltip.ts

```diff
@@ -1059,7 +1059,13 @@ class Tooltip {
                 if (
                     chart.polar ||
                     currentSeries.options.clip === false ||
-                    currentSeries.shouldShowTooltip(checkX, checkY)
+                    (
+                        points.length === 1 ?
+                            currentSeries.shouldShowTooltip(checkX, checkY) :
```

Comment:
Can we optimize this by always going the `points.some` route? In other words, are there any cases where `points.length === 1` and `currentSeries` is not the same as `points[0].series`?